src/main/java/com/duke/Duke.java

```diff
@@ -0,0 +1,197 @@
+package com.duke;
+
+import com.file.FileManager;
+import com.task.Deadline;
+import com.task.Event;
+import com.task.Task;
+import com.task.Todo;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.io.IOException;
+
+public class Duke {
+
+    public static FileManager fileTask;
+    static ArrayList<Task> list = new ArrayList<>();
+    public static String[] dukeLines = new String[100];
+    public static int dukeLineCount = 0;
+    public final static String LINESEPARATER =  "\t______________________________________________________________________";
+
+    public static void addTask(Task item) {
+        list.add(item);
+    }
+    private boolean active = false;
+
+    public Duke(){
+        active = true;
+        printLine();
+        System.out.println("\tHello from");
+        System.out.println("\t ____        _        ");
+        System.out.println("\t| _ \\ _   _| | _____ ");
+        System.out.println("\t| | | | | | | |/ / _ \\");
+        System.out.println("\t| |_| | |_| |   <  __/");
+        System.out.println("\t|____/ \\__,_|_|\\_\\___|");
+        loadFromFile();
+        endLine();
```

Comment:
These lines are currently not of the same level  of abstraction. You could [apply SLAP](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) here.